R/outliers.R

```diff
@@ -128,6 +128,11 @@ detect_outlr = function(x = seq_along(y), y,
 #'   `y`).
 #' @param y Signal values.
 #' @param n Number of time steps to use in the rolling window. Default is 21.
+#'   This value is centrally aligned. When `n` is an odd number, the
+#'   rolling range goes between `(n-1)/2` `time_value`s before to `(n-1)/2`
+#'   `time_value`s after. When `n` is even, then  the
+#'   rolling range goes between `n/2-1` `time_value`s before to `n/2`
+#'   `time_value`s after.
```

Comment:
If this is based on sliding, it's probably not quite accurate; slide is based on a sliding time window/range, which, if the `time_value`s are not evenly spaced, could contain a variable number of `time_value`s.  (Separate issue: I wish we had some better / less easily confused terminology to use, but even in slide docs, it might be a bit easy to confuse the two.)

Reply:
I think the current way we would describe this is "time steps before" and "time steps after".  Not much better, but good for consistency.

Reply:
Done.